index.html

```diff
@@ -125,10 +125,17 @@ <h2>Scope</h2>
           the context of HTML.</p>
         <p>The Group SHOULD:</p>
         <ol class='todo'>
-          <li>Publish a Recommendation for a new <a href="https://www.w3.org/TR/ttml2/">Timed
-              Text Markup Language 2 (TTML2)</a> specification. In the process
-            of producing this new revision, the Group SHOULD:
+          <li>Publish a new <a href="https://www.w3.org/TR/ttml3/">Timed
+              Text Markup Language 3 (TTML3)</a> specification, consisting of 
+              a series of modules where each module and any base document
+              is a Recommendation. In the process
+              of producing this new revision, the Group SHOULD:
             <ol type="a">
+              <li>Include a framework defining how each module integrates with
```

Comment:
Change "each module integrates" to "modules integrate".

Reply:
Done.